### PR TITLE
Change environment variables to function variables

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

Change references of environment variables to function variables after https://github.com/open-runtimes/open-runtimes/pull/91

## Test Plan

None

## Related PRs and Issues

* https://github.com/appwrite/docs/pull/248

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: app/views/docs/functions.phtml [hunk 1]
```diff
@@ -79,7 +79,7 @@ return function ($req, $res) {
 
   $secretKey =
     $req['env']['SECRET_KEY'] ?:
```
Comment (gewenyu99) [inline app/views/docs/functions.phtml:81]: You missed one!

File: app/views/docs/functions.phtml [hunk 2]
```diff
@@ -330,7 +330,7 @@ public RuntimeResponse main(RuntimeRequest req, RuntimeResponse res) throws Exce
 
     String secretKey = env.containsKey("SECRET_KEY")
```
Comment (gewenyu99) [inline app/views/docs/functions.phtml:331]: And here :P

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: app/views/docs/functions.phtml (hunk 1 above)
Comment (stnguyen90) [reply app/views/docs/functions.phtml]: updated! thanks for catching that! 🙏

File: app/views/docs/functions.phtml (hunk 2 above)
Comment (stnguyen90) [reply app/views/docs/functions.phtml]: updated! thanks for catching that! 🙏

**Review by gewenyu99 — APPROVED** [review]

**Pull request merged**